src/main/java/frc/robot/RobotContainer.java

```diff
@@ -46,187 +47,211 @@
+                                                new SwerveModuleIoReplay());
+                                vision = Constants.HAVE_VISION
+                                                ? new Vision(driveBase.getPoseEstimator(), new VisionIoSimAndReplay())
+                                                : null;
+                                break;
+
+                        default:
+                                throw new RuntimeException("Unknown Run Mode: " + Constants.CURRENT_OPERATING_MODE);
+                }
+
+                // ----- Initialize Subsystems without Simulation and/or Log Replay Modes -----
+                aimer = Constants.HAVE_AIMER ? new Aimer() : null;
+                colorSensor = Constants.HAVE_COLOR_SENSOR ? new ColorSensor() : null;
+                feeder = Constants.HAVE_FEEDER ? new Feeder() : null;
+                flywheel = Constants.HAVE_FLYWHEEL ? new Flywheel() : null;
+                intake = Constants.HAVE_INTAKE ? new Intake() : null;
+                // We can safely emit LED instructions even if there are no LEDs.
+                // (The LED control hardware is built into the RoboRio so always "exists".)
+                leds = new Leds();
+
+                // ========================= Autonomous =========================
+                // ---------- Create Named Commands for use by Path Planner ----------
+                NamedCommands.registerCommand("Spin 180",
+                                DriveCommands.spinCommand(driveBase, Rotation2d.fromDegrees(180), 1));
+                NamedCommands.registerCommand("StartIntake", LedCommands.blinkCommand(leds, Color.kPurple));
+                if (Constants.HAVE_FLYWHEEL) {
+                        NamedCommands.registerCommand("Spin Up Flywheel",
+                                        ShooterCommands.spinUpFlywheelCommand(flywheel));
+                }
+
+                Command aimCommand = new ConditionalCommand(
+                                // Turn to Blue Speaker.
+                                DriveCommands.turnToTargetCommand(driveBase,
+                                                Constants.BLUE_SPEAKER_LOCATION, 4.5),
+                                // Turn to Red Speaker.
+                                DriveCommands.turnToTargetCommand(driveBase,
+                                                Constants.RED_SPEAKER_LOCATION, 4.5),
+                                () -> DriverStation.getAlliance().get() == DriverStation.Alliance.Blue);
+                Command autoShootCommand;
+                if (Constants.HAVE_SHOOTER) {
+                        autoShootCommand = ShooterCommands.shootCommand(flywheel, feeder, leds, colorSensor);
+                } else {
+                        autoShootCommand = LedCommands.blinkCommand(leds, Color.kOrange);
+                }
+                NamedCommands.registerCommand("ShootNote", new SequentialCommandGroup(aimCommand, autoShootCommand));
```

Comment:
I think this needs to be `Auto Shoot` based on the changes that Gemma and Paul made.